test/remote/oauth_tests.js

```diff
@@ -16,7 +16,7 @@ const PUBLIC_CLIENT_ID = '3c49430b43dfba77';
 const MOCK_CODE_VERIFIER = 'abababababababababababababababababababababa';
 const MOCK_CODE_CHALLENGE = 'YPhkZqm08uTfwjNSiYcx80-NPT9Zn94kHboQW97KyV0';
 
-describe('/oauth/ routes', function () {
```

Comment:
lol, this was actually on purpose because it's decsribing "the oauth routes" rather than any particular route; maybe should have said "/oauth/* routes" or similar :-)

Reply:
@rfk my bad!